_pages/clients/python.md

```diff
@@ -64,206 +60,164 @@ client = Algorithmia.client("YOUR_API_KEY", "https://mylocalendpoint")
-import Algorithmia
-from Algorithmia.acl import ReadAcl, AclType
-
-apiKey = "YOUR_API_KEY"
-# Create the Algorithmia client
-client = Algorithmia.client(apiKey)
+# Download contents of file as a string
+if client.file(response.result).exists() is True:
+    output = client.file(response.result).getBytes()
+{% endhighlight %}
 
-# Set your Data URI
-nlp_directory = client.dir("data://YOUR_USERNAME/nlp_directory")
-# Create your data collection if it does not exist
-if nlp_directory.exists() is False:
-    nlp_directory.create()
+This will get the image as binary data, saving it to the variable `input`, which might be useful when writing algorithms that are part of an image processing pipeline. For more image-manipulation tutorials, see the [Computer Vision Recipes]({{site.baseurl}}/tutorials/recipes/#computer-vision). 
 
-# Create the acl object and check if it's the .my_algos default setting
-acl = nlp_directory.get_permissions()  # Acl object
-acl.read_acl == AclType.my_algos  # True
+If the file was text (an image, etc), you could instead use the function `.getString()` to retrieve the file's content as a string. For more methods on how to get a file using the Data API from a data collection go to the [API Specification](http://docs.algorithmia.com/#getting-a-file).
 
-# Update permissions to private
-nlp_directory.update_permissions(ReadAcl.private)
-nlp_directory.get_permissions().read_acl == AclType.private # True
 
-text_file = "data://YOUR_USERNAME/nlp_directory/jack_london.txt"
+Additional Functionality
+------------------------
 
-# Upload local file
-if client.file(text_file).exists() is False:
-    client.file(text_file).putFile("/your_local_path_to_file/jack_london.txt")
+In addition to the functionality covered in this guide, the Python Client Library provides a complete interface to the Algorithmia platform, including managing Algorithms (and their versions and builds), organizations, users, source control, and the additional functionality of the Data API. You can find all the details, including Python examples, in the API Docs.
 
-# Download contents of file as a string
-if client.file(text_file).exists() is True:
-    input = client.file(text_file).getString()
-
-# Create the algorithm object using the Summarizer algorithm
-algo = client.algo("nlp/Summarizer/0.1.3")
-# Pass in input required by algorithm
-try:
-    # Get the summary result of your file's contents
-    print(algo.pipe(input).result)
-except Exception as error:
-    # Algorithm error if, for example, the input is not correctly formatted
-    print(error)
-{% endhighlight %}
+Next Steps
+----------
 
+If you're a developer looking to call algorithms from the marketplace or those created by your team members, you're ready to start coding! If you're a data scientist or developer who will be building and deploying new algorithms, you can move on to the [Algorithm Development > Getting Started](/developers/algorithm-development/your-first-algo/) guide.
```

Comment:
```suggestion
If you're a data scientist or developer who will be building and deploying new algorithms, you can move on to the [Algorithm Development > Getting Started](/developers/algorithm-development/your-first-algo/) guide. 
```

Reply:
Just removing mention of marketplace, but I'll think of some better messaging here